### PR TITLE
Adding simple <html> opening tag

### PR DESCRIPTION
**Pull request by romxz**

### CODE REVIEW

File: snippets/html.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
 			"<!--[if lt IE 7]>      <html class=\"no-js lt-ie9 lt-ie8 lt-ie7\"> <![endif]-->",
```
Comment (sidthesloth92) [inline snippets/html.json:6]: There is already an html opening tag here!

**Comment by sidthesloth92** [comment]
The `html` tag is already present in the first line.

File: snippets/html.json (hunk 1 above)
Comment (romxz) [reply snippets/html.json:6]: Yep. It was more of out of the convenience to have the simple <html> tag (similar to another vscode extension that I was using instead). More visually clear for a repetitive task, but obviously not a big deal.

**Comment by sidthesloth92** [comment]
@all-contributors please add @romxz for code.

**Pull request merged**